dockerfiles/csharp/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.10-alpine
+WORKDIR /go/src/github.com/sourcegraph/lsp-adapter
+COPY . .
+RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-adapter
+
+# TODO Try reducing the image size by using https://github.com/frol/docker-alpine-mono instead of ubuntu.
+# RUN apk add mono --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted
+# RUN apk add --no-cache nodejs nodejs-npm tini bash
+
+FROM ubuntu:16.04
+RUN apt-get update
```

Comment:
Read https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

> Always combine RUN apt-get update with apt-get install in the same RUN statement
> Using apt-get update alone in a RUN statement causes caching issues and subsequent apt-get install instructions fail

Also, in their recommended example they put all packages in a single install statement. It reduces the number of cacheable layers, but it also reduces the number of layers, which is important if this is the final stage and not the build stage.

Reply:
Interesting, thanks for pointing this out! I added `apt-get update` to every `apt-get install` command now.